web-common/src/components/search/Search.svelte

```diff
@@ -25,17 +25,15 @@
   });
 </script>
 
-<form class="relative w-full px-1">
+<form class="relative w-full">
   <div class="flex absolute inset-y-0 items-center pl-2 ui-copy-icon">
     <Search />
   </div>
   <input
     bind:this={ref}
     type="text"
     autocomplete="off"
-    class="outline-none bg-gray-100 surface-impression border border-gray-200 dark:border-gray-400
-        rounded-sm focus:border-gray-300
-        block w-full pl-8 p-1"
+    class="bg-white border border-gray-200 focus:border-blue-400 outline-none rounded-sm block w-full pl-8 p-1"
```

Comment:
These changes are affecting search components at other places

Reply:
Ah, good catch, just fixed the padding in 3 other Search instances. Janet created this [new Search component](https://www.figma.com/file/nqqazRo1ckU9ooC9ym9weI/Rill-Design-System?type=design&node-id=78-203&mode=design&t=sGyyC9hgxQtE13gy-0) that this code moves towards.

Reply:
Note that the blue outline is from the focus state, and these search instances are currently set to focus-on-mount.